packages/python-packages/apiview-copilot/README.md

```diff
@@ -67,6 +61,27 @@ If you would like to search the knowledge base and see the output the way the LL
 
 This will dump the results to context.md which you can then view in VSCode with the preview editor.
 
+## Getting Comments from APIView
+
+If you need to retrieve comments from APIView, you can use the following command:
+
+`cli.bat apiview get-comments --review-id <ID> [--environment "production"|"staging"]`
+
+This command retrieves comments from APIView for a specific review ID. You can specify the environment (production or staging) to get the comments from the appropriate APIView instance, but the default is production.
+
+If you need RBAC permissions to access CosmosDB, you can run the following script:
```

Comment:
can we also mention the right commands + subscription for cli auth before running the script, something like:
```
az account set --subscription a18897a6-7e44-457d-9260-f2854c0aca42
az login --tenant <your-tenant-id>
```